Header header
int32   frame_height
int32   frame_width
float32 inference_time
int32 	no_of_lanes
int32[]  lane_coordinates
int32[]  class_ids